src/index.js

```diff
@@ -502,19 +565,19 @@ function writeGroupsSidebarNOIMaps(ODHdata) {
 			var temp = objects[groupName];
 			temp['name'] = groupName;
 			sorted[orderN] = temp;
-
+			
 		}
 
 		for(var index in sorted) {
 			let categoryGroupClone = categoryGroup.clone().removeAttr("style");
 			////console.log(objects);
 			let image = '';
 			if(typeof sorted[index].image !== 'undefined'){
-				image = '<img src="'+replaceImageUrl(sorted[index].image)+'" />';
+				image = '<img src="'+sorted[index].image+'" />';
```

Comment:
We need the replacement here for testing/staging purposes... please do not touch this